internal/pkg/cli/deploy/workload.go

```diff
@@ -361,12 +363,18 @@ func (d *workloadDeployer) uploadContainerImages(out *UploadArtifactsOutput) err
 	if len(buildArgsPerContainer) == 0 {
 		return nil
 	}
-	err = d.repository.Login()
+	uri, err := d.repository.Login()
 	if err != nil {
 		return fmt.Errorf("login to image repository: %w", err)
 	}
 	out.ImageDigests = make(map[string]ContainerImageIdentifier, len(buildArgsPerContainer))
 	for name, buildArgs := range buildArgsPerContainer {
+		buildArgs.URI = uri
+		buildArgsList, err := buildArgs.GenerateDockerBuildArgs(dockerengine.New(exec.NewCmd()))
+		if err != nil {
+			return fmt.Errorf("generate docker build args: %w", err)
+		}
+		log.Infof("Building your container image: docker %s\n", strings.Join(buildArgsList, " "))
```

Comment:
Should this use `spinner`?

Reply:
Even though we are printing the build label over here. In the next PR , this build label will passed to [LabeledSyncBuffer](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/term/syncbuffer/syncbuffer.go#L53) struct and then Print function will handle the logging.

Reply:
I see. Can you add `// TODO(adi) handle this log in syncBuffer's Print function` here? Personally, I found this useful for two reasons:
1. Remind us that we still need to do this
2. In the next PR, the reviewer will see this "TODO" being removed, along with "handle this log in syncBuffer's Print function". This gives them some background information of the relation between that PR and this PR.